dev/com.ibm.ws.microprofile.reactive.streams.operators_fat_tck/publish/tckRunner/tck/src/test/java/org/eclipse/microprofile/reactive/streams/operators/tck/arquillian/ReactiveStreamsArquillianTck.java

```diff
@@ -0,0 +1,200 @@
+/*******************************************************************************
+ * Copyright (c) 2018, 2022 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
From the [Apache license](https://www.apache.org/licenses/LICENSE-2.0):
> - You must cause any modified files to carry prominent notices stating that You changed the files; and
> - You must retain, in the Source form of any Derivative Works that You distribute, all copyright, patent, trademark, and attribution notices from the Source form of the Work, excluding those notices that do not pertain to any part of the Derivative Works; and

I think that means
- We need to add something prominent to say we've changed this file
- We can't change the copyright header

Not sure if that means we add our own copyright header alongside this one, as well as a notification of what we've modified?

Reply:
When I did this previously for an Apache class I was asked to added this which the lawyers accepted:
```
/**
 * This is a derivative work based on Apache Commons-Validator 1.7 UrlValidator and DomainValidator classes
 * https://commons.apache.org/proper/commons-validator/xref/org/apache/commons/validator/routines/UrlValidator.html
 * https://commons.apache.org/proper/commons-validator/xref/org/apache/commons/validator/routines/DomainValidator.html
 *
 * Changelog
 * Combine UrlValidator and DomainValidator to simplify URL validation
 * Remove IANA TLD Checking
 * Remove Country Checking
 * Simplify options to Local only change behaviour from local and SVC
 * Remove file URL type
 * Add SVC as a particular tld that requires
 *
 * Validate URLs
 */
 ```
 
Copyright header stayed as-is without IBM attribution - but the vast majority of my changes were to remove a chunks of the original classes and the above block was added above the class declaration